src/auth/index.ts

```diff
@@ -0,0 +1,83 @@
+import express, { Request } from 'express'
+import cors from 'cors'
+import * as yup from 'yup'
+import * as jwt from 'jsonwebtoken'
+
+import { table, client } from '../index'
+
+export const router = express.Router()
+
+router.use(cors(
+    {
+        credentials: true
+    }
+))
+router.use(express.json())
+
+router.get('/', (req, res) => {
+    res.json({
+        message: '🗝'
+    })
+})
+
+router.post('/login', async function (req, res, next) {
+    const isValidInput = await validateLogin(req.body).catch(err => {
+        res.status(401).json({
+            message: err.message
+        })
+
+    })
+
+    await table
```

Comment:
@Alexandria Just curious, is this `await` necessary?

Reply:
After looking at it and running through some tests...I am not sure. It seems to work either way.
My thought process is to wait until I know that the connection to the  pg `Client` has been made and then query the tables.